src/main/scala/org/ergoplatform/modifiers/mempool/ErgoTransaction.scala

```diff
@@ -120,6 +124,27 @@ case class ErgoTransaction(override val inputs: IndexedSeq[Input],
       .validate(txManyInputs, inputs.size <= Short.MaxValue, s"$id: ${inputs.size}")
       .validate(txManyDataInputs, dataInputs.size <= Short.MaxValue, s"$id: ${dataInputs.size}")
       .validate(txManyOutputs, outputCandidates.size <= Short.MaxValue, s"$id: ${outputCandidates.size}")
+      .validateSeq(outputCandidates)((validationState, out) => validationState
+        .validate(txMaxBoxSize, getByteLength[ErgoBox](out.toBox(id, 0), ErgoBoxSerializer) <= ErgoConstants.MaxBoxSize.get)
+        .validate(txAdditionalRegistersCount, out.additionalRegisters.size <= ErgoBox.nonMandatoryRegistersCount)
+        .validateSeq(out.additionalRegisters)((validationState, reg) => validationState
+          .validate(txRegisterData, reg._2 match {
+            case (coll: ConcreteCollection[_]) =>
+              val w = SigmaSerializer.startWriter()
+              ValueSerializer.serialize(coll, w)
+              w.length <= ErgoConstants.MaxByteArrayLength.get
+            case (constant: Constant[_]) =>
+              constant.tpe match {
+                case l: SBigInt.type =>
+                  val w = SigmaSerializer.startWriter()
+                  DataSerializer.serialize(constant.value, constant.tpe, w)
+                  w.length <= ErgoConstants.MaxBigIntSizeInBytes.get
```

Comment:
Same as above

Reply:
It's not an `Option` or `Try`, it is `SizeConstant` class that always can run `.get`